demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -218,15 +218,19 @@ def _search_packs_by_items_names(items_names: Union[str, list],
     @staticmethod
     def _search_packs_by_items_names_or_ids(items_names: Union[str, list],
                                             items_list: list,
-                                            exclude_ignored_dependencies: bool = True) -> set:
+                                            exclude_ignored_dependencies: bool = True,
+                                            incident_or_indicator: Optional[str] = 'both') -> set:
         """
         Searches for implemented packs of the given items.
 
         Args:
             items_names (str or list): items names to search.
             items_list (list): specific section of id set.
             exclude_ignored_dependencies (bool): Determines whether to include unsupported dependencies or not.
-
+            incident_or_indicator (str):
+                'indicator' to search packs with indicator fields,
```

Comment:
In the condition you are checking for "Indicator" / "Incident" / "both" (Camal cased). Please update the docstring accordingly.
I also think you should turn "both" into "Both" for consistency.